dev/change_version.py

```diff
@@ -248,6 +249,24 @@ def change_project_number_Doxyfile(file, new_version):
     f.write(changed_str)
     f.close()
 
+def change_project_number_readme(file, new_version):
+    changed_str = ""
+
+    markers = ["<Reference", "<HintPath>"]
+    #                             0   .  15etc
+    version_regex = re.compile(r'[0-9]\.[0-9]+')
+
+    f = open(file, 'r')
+    for line in f:
+        if any(marker in line for marker in markers):
+            changed_str += version_regex.sub(new_version, line, count=1)
+        else:
+            changed_str += line
+    f.close()
+
+    f = open(file, 'w')
+    f.write(changed_str)
+    f.close()
```

Comment:
We're using a standard version format, so let's not suggest regexes which will not handle it well :-)